README.md

```diff
@@ -13,8 +13,8 @@
 [doc-stable-url]: https://control-toolbox.org/control-loss/stable/
 
 This repo is part of the [control-toolbox ecosystem](https://github.com/control-toolbox). 
-The control-toolbox ecosystem gathers Julia packages for mathematical control and applications. The root package is [OptimalControl.jl](https://github.com/control-toolbox/OptimalControl.jl) which aims to provide tools to modelise and solve optimal control problems by direct and indirect methods. [![doc OptimalControl.jl](https://img.shields.io/badge/doc-OptimalControl.jl-blue)](http://control-toolbox.org/OptimalControl.jl)
+The control-toolbox ecosystem gathers `Julia` packages for mathematical control and applications. The root package is [`OptimalControl.jl`](https://github.com/control-toolbox/OptimalControl.jl) which aims to provide tools to modelise and solve optimal control problems by direct and indirect methods. [![doc OptimalControl.jl](https://img.shields.io/badge/doc-OptimalControl.jl-blue)](http://control-toolbox.org/OptimalControl.jl)
```

Comment:
Actually, instead of `OptimalControl.jl` we simply write OptimalControl.jl.

Reply:
Same for Julia. Be careful since, it seems that you have overwritten corrections I have made before.

Reply:
This issue is solved.